CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Ctrl-C no longer kills background jobs for which job control is disabled, matching POSIX semantics (#6828).
 - Improve Gradle completion
 - Fixed `pushd`'s behavior with respect to the directory stack when given an invalid argument
+- A new variable, `$fish_force_vi_cursor`, has been added. This can be set to force `fish_vi_cursor` to attempt changing the cursor shape in vi mode, regardless of terminal.
```

Comment:
Now the name here and in docs is out of sync.

Reply:
Fixed; also added a note about the option removal.